### ISSUE TITLE
production.html + development.html don't renders right Only SSR works when using feathersjs

### ISSUE THREAD

**Issue by frank-dspeed** [donejs/donejs#972]
__How often can you reproduce it?__ 

- [X ] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__
[Thu Jun 22 2017 15:51:57 GMT+0100 (WEST)] "GET /" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:01 GMT+0100 (WEST)] "GET /production.html" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:01 GMT+0100 (WEST)] "GET /dist/steal.production.js" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:01 GMT+0100 (WEST)] "GET /dist/bundles/artist/index.js" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:02 GMT+0100 (WEST)] "GET /dist/bundles/artist/index.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:02 GMT+0100 (WEST)] "GET /dist/bundles/artist/list/list.js" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:02 GMT+0100 (WEST)] "GET /dist/bundles/artist/list/list.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"
[Thu Jun 22 2017 15:52:11 GMT+0100 (WEST)] "GET /dist/bundles/artist/index.css" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0"

```
<head>
  <title>artist</title>
<link type="text/css" rel="stylesheet" href="http://localhost/dist/bundles/artist/index.css"><link type="text/css" rel="stylesheet" href="http://localhost/dist/bundles/artist/list/list.css"></head>
<body>
  <script src="./dist/steal.production.js"></script>
 

</body>
```
resulting html contains the right css after steal load and that but its missing h1 message tag
and missing one component that should even show a loading.component
so i say stache don't gets rendered after build but befor in donejs develop all renders right.

data gets requested and returned from api endpoint but nothing renders.

__Steps to reproduce:__
https://github.com/direktspeed/prove-bugs
please see 
https://github.com/direktspeed/prove-bugs/issues/1

__Expected results:__
showing same result as donejs develop but request less files

__Actual results:__
requesting less files renders nothing after steal load dom don't updates.

__Environment:__

| Software                | Version
| ------------------ | -------
| donejs -V               |  1.0.0
| node -v                  |  7.10
| npm -v                   | 4.2.0
| Browser                 | newst firefox + chrome
| Operating system | ubuntu 16.10

**Comment by frank-dspeed** [comment]
https://github.com/donejs/bitcentive/issues/309

i think here we have the same issues over all as its using feathersJS

**Comment by matthewp** [comment]
Appears to be the same issue as in #971, so let's keep the discussion there.

**Comment by frank-dspeed** [comment]
i don't think its really the same issue i opened this only for fixing docs of steal-socket.io and or done-serve because many users will run into that and stop using donejs befor even understanding what can-zone is or done-autorender. For example if they copy a receipe using socket.io with steal-socket.io they are doomed !

example receips from canjs that get ported to donejs using socket.io exist :) even the tip to use steal-socket.io exists in guides.

the only fact why bitballs and other examples work are the usage of ssr via donejs middelware for express. so you can't even deploy production builds on diffrent servers

- #972 is more about can-zone can't be used with socket.io canjs/can-zone#108
- #971 is about steal-socket.io stealjs/steal-socket.io#27

but your right now as i know about can-zone we need to document that also! ASAP!

both demonstrated via direktspeed/prove-bugs#1

**Comment by frank-dspeed** [comment]
https://github.com/donejs/autorender/pull/68

Closes: #972 in reality
